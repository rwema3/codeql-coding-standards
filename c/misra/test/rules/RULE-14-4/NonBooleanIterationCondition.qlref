rules/RULE-14-4/NonBooleanIterationCondition.ql