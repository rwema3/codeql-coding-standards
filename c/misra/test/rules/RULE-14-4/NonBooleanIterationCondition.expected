| test_iteration.c:5:20:5:20 | i | Iteration condition has non boolean type int. |
| test_iteration.c:7:10:7:11 | l1 | Iteration condition has non boolean type int. |
